lib/pharos/salt.rb

```diff
@@ -27,6 +27,11 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Call mine.update on salt for every minion specified by targets.
+    def self.mine_update(targets: "*")
```

Comment:
Right. But I'd like to avoid adding a new method, which is simply an alias for `call`, for every salt function that we might end up calling. Is it really that different from a user perspective to say "I want to call mine_update", than "I want salt to call mine.update" ?

I'd be in favor of adding a function `all` which would be an alias of `call` for targets='*'. Then your call would be:

```ruby
Pharos::Salt.all("mine.update")
```

Another possible option would be to add a function like this:

```ruby
def self.perform(action, target = "*")
    # ...
end
```

This `action` parameter would expect a symbol `:mine_update` or any other allowed function, and it would do the translation from this symbol to the proper salt function. Then your call would've been:

```ruby
Pharos::Salt.perform(:mine_update)
```

And this would work for similar functions. That being said, this abstraction would not probably translate well whenever we want to add parameters 😅 What do you think ?